src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -195,6 +195,7 @@ func runHyperKit(args []string) {
 	if err != nil {
 		log.Fatalln("Error creating hyperkit: ", err)
 	}
+	h.SetLogger(log.StandardLogger())
```

Comment:
`h.SetLogger` turns into a call to `SetLogger` which is global in the `hyperkit` package, maybe it would be better to call that from a `func init()`?

The `h.SetLogger` wrapper itself is a bit odd since the state is global not per-`h`. Perhaps we should nuke it?